src/style.css

```diff
@@ -0,0 +1,134 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Alkatra", cursive;
+}
+
+body {
+  font-size: 16px;
+  background-color: rgb(216, 227, 236);
+}
+
+ul {
+  list-style: none;
+}
+
+button {
+  cursor: pointer;
+  border: none;
+  color: #989898;
+  transition: 300ms;
+}
+
+button:hover {
+  color: black;
+  transform: scale(1.1);
+}
+
+input {
+  outline: none;
+  border: none;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#container {
+  max-width: 400px;
+  width: 100%;
+  height: auto;
+  background-color: #fff;
+  border: 1px solid #989898;
+  margin-top: 50px;
+  box-shadow:
+    inset 0 -3em 3em rgba(0, 0, 0, 0.1),
+    0 0 0 1px rgba(152, 152, 152, 0.3),
+    0.3em 0.3em 1em rgba(0, 0, 0, 0.3);
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 5px;
+  border-bottom: #989898 solid 1px;
+}
+
+h1 {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.fa-arrows-rotate {
+  color: #989898;
+  padding-right: 5px;
+  font-size: 14px;
+}
+
+#frm-task {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: #989898 solid 1px;
+}
+
+#new-task {
+  color: #989898;
+  width: 100%;
+  padding: 5px;
+}
+
+.fa-square-check {
+  color: #989898;
+  padding-right: 8px;
+}
+
+#ul-tasks {
+  display: flex;
+  flex-direction: column;
```

Comment:
[optional]Please, kindly consider to use a class related to the display flex property, you can use that clase instead of write display flex several times.

Reply:
Hi @oovillagran, ✋  I create a class to refactor the use of `flex`:

https://github.com/LarryIVC/ToDoList-Reviewe/blob/4352e6ecd909eabde8e6f3743637d594b6e55015/src/style.css#L34

And use in the `HTML`:

https://github.com/LarryIVC/ToDoList-Reviewe/blob/4352e6ecd909eabde8e6f3743637d594b6e55015/src/index.html#L20

And JavaScript code:

https://github.com/LarryIVC/ToDoList-Reviewe/blob/4352e6ecd909eabde8e6f3743637d594b6e55015/src/logic.js#L27

Thanks for your observations. 😄